### PR TITLE
GUI temp

### PR DESCRIPTION
**Pull request by timjon**

### CODE REVIEW

**Review by Sebastian-Fransson — APPROVED** [review]

File: F.R.O.S.T/app/src/main/java/t/s/o/r/f/frost/MainActivity.java [hunk 1]
```diff
@@ -65,6 +66,8 @@ protected void onCreate(Bundle savedInstanceState) {
         tv = (TextView) findViewById(R.id.collision_text); //TextView for collision text.
         v = findViewById(R.id.cc_view4);
         tvbg = findViewById(R.id.text_background);
+        fireImage = (ImageView)findViewById(R.id.imageView1);
```
Comment (IsabelleTornqvist) [inline F.R.O.S.T/app/src/main/java/t/s/o/r/f/frost/MainActivity.java:69]: Maybe have a space here? If they are not related to the stuff above. Maybe a comment/description

**Review by IsabelleTornqvist — APPROVED** [review]

**Review by Anthony1911 — CHANGES_REQUESTED** [review]
Add explanation for this section

File: F.R.O.S.T/app/src/main/res/layout-land/activity_main.xml [hunk 2]
```diff
@@ -222,4 +222,12 @@
         <item android:name="android:src">@drawable/img_background</item>
     </style>-->
 
+    <ImageView
+        android:id="@+id/imageView1"
```
Comment (pontuslae) [inline F.R.O.S.T/app/src/main/res/layout-land/activity_main.xml:226]: Sorry what is this imageview1????????
I'm stupid and don't understands.

File: F.R.O.S.T/app/src/main/java/t/s/o/r/f/frost/MainActivity.java [hunk 3]
```diff
@@ -539,6 +542,11 @@ static void handleInput(){
                     displayTemp(value);
                     break;
                 case 'f': //Flame sensor input
+                    if(s.charAt(3) == '1'){
```
Comment (pontuslae) [inline F.R.O.S.T/app/src/main/java/t/s/o/r/f/frost/MainActivity.java:545]: This entire if statement just be, but It's just a suggestion.
fireImage.setVisibility(s.charAt(3) == '1' ? View.VISIBLE : View.INVISIBLE);

**Review by Anthony1911 — APPROVED** [review]

**Pull request merged**